changelog.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 0.3.10
+
+Creates new Debian package `securedrop-keyring` for managing the SecureDrop
+Release Signing Key. Rotates the signing key currently in use by setting
+a dependency on the other Debian packages, so that currently running deployments
+will have their apt keyrings updated via automatic nightly updates.
```

Comment:
Not a bad edit at all, but let's stick with what's in place given time constraints. In other messaging surrounding the release, such as blog post announcement, we can be explicit about the specifics.